app/controllers/idv/session_errors_controller.rb

```diff
@@ -7,35 +7,48 @@ class SessionErrorsController < ApplicationController
     before_action :confirm_idv_session_step_needed
     before_action :set_try_again_path, only: [:warning, :exception]
 
-    def exception; end
+    def exception
+      log_event
+    end
 
     def warning
-      @remaining_attempts = Throttle.new(
+      throttle = Throttle.new(
         user: effective_user,
         throttle_type: :idv_resolution,
-      ).remaining_count
+      )
+
+      @remaining_attempts = throttle.remaining_count
+      log_event based_on_throttle: throttle
     end
 
     def failure
-      @expires_at = Throttle.new(
+      throttle = Throttle.new(
         user: effective_user,
         throttle_type: :idv_resolution,
-      ).expires_at
+      )
+      @expires_at = throttle.expires_at
+      log_event based_on_throttle: throttle
```

Comment:
Is there a reason/convention to not use parentheses around the method arg here?

Reply:
I have become overly reliant on a linter yelling at me about things like this :-). I agree parens would be helpful here.